### ISSUE TITLE
Define Bo3 Map order without Veto

### ISSUE THREAD

**Issue by Iwhite67** [PhlexPlexico/G5V#150]
Hello, actualy Create match vue have possibility to order a bo3 map when veto are disabled ?

**Comment by PhlexPlexico** [comment]
Yes, just select the maps you wish to play in the order you wish to play them. It sends the structure in an ordered manner (well, it should).

**Comment by Iwhite67** [comment]
Ok thanks



________________________________
